packages/db/src/core/integration/file-url.ts

```diff
@@ -73,8 +73,8 @@ export function fileURLIntegration(): AstroIntegration {
 					}
 					await Promise.all(unlinks);
 					const assetDir = new URL(config.build.assets, config.outDir);
-					const assetFiles = await fs.promises.readdir(assetDir);
-					if (!assetFiles.length) {
+					const assetFiles = await fs.promises.readdir(assetDir).catch(() => []);
```

Comment:
this was also causing an issue if I didn't have an assets folder.

@bholmesdev I'm surprised that this integration is responsible for cleaning up the `_astro` folder? I'd assume this is astro's responsibility and already handled.

Reply:
nvm ignore me, fixed this a different way